Dockerfile

```diff
@@ -1,9 +1,9 @@
 # syntax=docker/dockerfile-upstream:master
 
 ARG RUNC_VERSION=v1.2.3
-ARG CONTAINERD_VERSION=v2.0.0
+ARG CONTAINERD_VERSION=v2.0.1
```

Comment:
This will conflict with the other (runc) one; perhaps a good opportunity to add a comment in between